package/boot/uboot-mvebu/Makefile

```diff
@@ -16,6 +16,8 @@ PKG_HASH:=f54baf3f9325bf444c7905f3a5b6f83680edb1e6e1a4d5f8a5ad80abe885113f
 include $(INCLUDE_DIR)/u-boot.mk
 include $(INCLUDE_DIR)/package.mk
 
+UBOOT_MAKE_FLAGS:=
+
```

Comment:
Why is this needed?

Reply:
Douring build we get more than 60 errors:
/bin/sh: HOSTCPPFLAGS: command not found

Above change will remove those errors. Several other uboot Makefiles use this, examples are uboot-ar71xx and uboot-lantiq